in_toto/verifylib.py

```diff
@@ -672,9 +663,7 @@ def verify_match_rule(rule, source_artifacts_queue, source_artifacts, links):
   try:
     dest_link = links[dest_name]
   except KeyError:
-    raise RuleVerificationError("Rule '{rule}' failed, destination link"
-        " '{dest_link}' not found in link dictionary".format(
-            rule=" ".join(rule), dest_link=dest_name))
+    return source_artifacts_queue
```

Comment:
It seems to me that you had reverted this unwanted change in a previous version of this PR. Did you force push back in? (Please also leave in the fixme comment above (L576-584).

Reply:
I thought that I had done so when I forced pushed to get rid of the merge commit. I have since rectified my mistake and rebased accordingly.